dataclients/kubernetes/kube.go

```diff
@@ -374,53 +369,42 @@ func shuntRoute(r *eskip.Route) {
 	r.Backend = ""
 }
 
-func healthcheckRoute(healthy, reverseSourcePredicate bool) *eskip.Route {
-	logFilters := []*eskip.Filter{{
-		Name: filters.StatusName,
-		Args: []interface{}{http.StatusOK}},
-	}
-	if !healthy {
-		logFilters[0].Args = []interface{}{http.StatusServiceUnavailable}
-	}
-	// log if unhealthy or a debug loglevel
-	if healthy && !log.IsLevelEnabled(log.DebugLevel) {
-		logFilters = append(logFilters, &eskip.Filter{
-			Name: filters.DisableAccessLogName,
-			Args: []interface{}{200},
-		})
-	}
+func healthcheckRoutes(reverseSourcePredicate bool) []*eskip.Route {
+	template := template.Must(template.New("healthcheck").Parse(`
+		kube__healthz_up:   Path("/kube-system/healthz") && {{.Source}}({{.SourceCIDRs}}) -> {{.LogFilter}} status(200) -> <shunt>;
+		kube__healthz_down: Path("/kube-system/healthz") && {{.Source}}({{.SourceCIDRs}}) && Shutdown() ->  status(503) -> <shunt>;
```

Comment:
Why you have no Logfilter in the shutdown route?

Reply:
I guess because we want to see healthcheck logs during shutdown.
This re-implements what we had before, the `disableAccessLog(200)` was introduced by #1650

Reply:
Right now, I can observe the shutdown health checks by ALBs or NLB control plane. I don't really care about the "up" message in healthchecks. For me the more interesting logs are the "down" messages.

Reply:
Yes, this mutes access log for "up" unless debug level is enabled.

Reply:
I have renamed `LogFilter` to `DIsableAccessLog` to make it more clear